### PR TITLE
Improve style to match Figma

### PR DESCRIPTION
**Pull request by gonzalo-vazquez-cao**

### CODE REVIEW

File: src/Auth/LogIn/LogIn.test.tsx [hunk 1]
```diff
@@ -12,8 +12,22 @@ jest.mock('react-firebase-hooks/auth')
 const mockSignIn = mocked(useSignInWithEmailAndPassword, true)
 
 const Dashboard = () => <div>I am the dashboard</div>
+const CreateAccount = 'If you do not have an account, click here</a> to contact us and create it'
```
Comment (fergus-orbach) [inline src/Auth/LogIn/LogIn.test.tsx]: seems like this can be deleted

File: src/Auth/LogIn/LogIn.test.tsx [hunk 2]
```diff
@@ -12,8 +12,22 @@ jest.mock('react-firebase-hooks/auth')
...
 
 describe(LogIn, () => {
+  it('It allows for user creation', async () => {
```
Comment (fergus-orbach) [inline src/Auth/LogIn/LogIn.test.tsx]:
```suggestion
  it('allows for user creation', async () => {
```

no need to write `it` twice ;)

File: src/Auth/LogIn/LogIn.test.tsx [hunk 3]
```diff
@@ -61,7 +75,7 @@ describe(LogIn, () => {
         </MemoryRouter>,
       )
 
-      const loginButton = await screen.getByRole('button')
+      const loginButton = await screen.getByRole('button', { name: '' })
```
Comment (fergus-orbach) [inline src/Auth/LogIn/LogIn.test.tsx:77]: why did you add the extra `{ name: '' }` selector here - is it needed?

File: src/Auth/LogIn/Login.module.css [hunk 4]
```diff
@@ -1,12 +1,42 @@
...
+.pageContainer > p, h1 {
+    width: 100%;
+    text-align: center;
+    color: white;
+}
+
+.pageContainer a {
+    color: white;
+}
+
```
Comment (fergus-orbach) [inline src/Auth/LogIn/Login.module.css]: I think these can be deleted

**Review by fergus-orbach — DISMISSED** [review]
A couple of small details which you can either fix as part of this PR or make a new commit. Otherwise super nice work man!

File: src/Auth/LogIn/LogIn.test.tsx (hunk 3 above)
Comment (gonzalo-vazquez-cao) [reply src/Auth/LogIn/LogIn.test.tsx:77]: It is, there are two components with the 'button' role and when is still loading the accessible name is an empty string. I tried to give it a meaningful name but failed. I forgot to ask you about it this morning but either way it needs to include name when looking for the component or the test fails.

File: src/Auth/LogIn/Login.module.css (hunk 4 above)
Comment (gonzalo-vazquez-cao) [reply src/Auth/LogIn/Login.module.css]: All can be removed except for the link styling, this needs to be white as well.

**Review by fergus-orbach — DISMISSED** [review]

**Review by fergus-orbach — APPROVED** [review]
